libdns.go

```diff
@@ -43,7 +47,8 @@ import (
 
 // RecordGetter can get records from a DNS zone.
 type RecordGetter interface {
-	// GetRecords returns all the records in the DNS zone.
+	// GetRecords returns all the records in the DNS zone. DNSSEC-specific
+	// records may or may not be included in an implementation-defined way.
```

Comment:
Rationale: Most providers implement DNSSEC automatically (or not at all) rather than expecting users to manually sign their zone. Because of this, many providers don't include the DNSSEC records in their standard "get zone" API calls, even though they're technically just standard records in the zone.

Reply:
Well, it definitely feels weird if that code doesn't work doesn't it... I feel like that should work.

What if the provider's DNSSEC behavior can be configured? Somewhere, there is a concrete type that the user has configured (with API key, etc). By default, all DNSSEC gets filtered out by all providers -- that way, calls always work reliably. But if a provider wanted to enable it, it could be configurable.

(If we do this, we should document what DNSSEC records look like. Maybe even provide some boilerplate code for them or an `if` statement or something. What would that look like?)